New Liveness autoconfigure health endpoint (#2)
hmcts/health-spring-boot-starter: README.md

[![Build Status](https://travis-ci.com/hmcts/health-spring-boot-starter.svg?branch=master)](https://travis-ci.com/hmcts/health-spring-boot-starter)
[ ![Download](https://api.bintray.com/packages/hmcts/hmcts-maven/health-spring-boot-starter/images/download.svg) ](https://bintray.com/hmcts/hmcts-maven/health-spring-boot-starter/_latestVersion)

This is a Spring Boot _starter_ library whose purpose is to autoconfigure any custom HMCTS actuator health endpoints.

> Currently this library contains only the liveness _NOP_ health check needed for Kubernetes to properly operate without restarting the pod. 

Reviewer: ```suggestion
Currently this library contains only the liveness _NO-OP_ health check needed for Kubernetes to properly operate without restarting the pod. 
```

Author: sure

The intention for this library is for any extra organisationally customised health to be implemented here and auto-configured for the projects.
 

> Currently this module automatically adds the liveness health detail item which then can be access via `[managment]/health/liveness`

Reviewer: ```suggestion
Currently this module automatically adds the liveness health detail item which then can be access via `[management]/health/liveness`
```

Author: done

Where:

> - `[managment]` is where the actuator endpoints are configured. Typically in HMCTS we configure it to `/`

Reviewer: ```suggestion
- `[management]` is where the actuator endpoints are configured. Typically in HMCTS we configure it to `/`
```

Author: done
